### DISCUSSION TITLE
requires-python:>=3.7

### DISCUSSION THREAD

**Discussion by srideep3** [pradyunsg/furo#644] [Potential Issue] [answered]
I am on 3.11.1 but still keep getting this error.

**Comment by pradyunsg** [comment, marked as answer]
There isn't enough information here to help you in any way. Please see [this link](https://codeblog.jonskeet.uk/2010/08/29/writing-the-perfect-question/) which provides a lot of information on how to ask a question such that it is possible for someone else to help you and give you an answer. :)
